xapian-glib/xapian-query-parser.cc

```diff
@@ -379,6 +405,35 @@ xapian_query_parser_set_stopper (XapianQueryParser *parser,
 }
 
 /**
+ * xapian_query_parser_set_default_op:
+ * @parser: a #XapianQueryParser
+ * @op: a #XapianQueryOp value
+ *
+ * Set the operator to use to combine non-filter query items when no explicit
+ * operator is used.
+ * Since 1.4
```

Comment:
I copied that from elsewhere - will send a follow-up patch to fix all the existing "Since 1.x" instances.